src/main/flight/servos.h

```diff
@@ -41,7 +41,7 @@ enum {
     INPUT_GIMBAL_PITCH,
     INPUT_GIMBAL_ROLL,
     INPUT_SOURCE_COUNT
-} inputSource_e;
```

Comment:
is this related to this PR?

Reply:
Yes. Making this a definition will trigger an error with `-fno-common`.

Reply:
I think it's missing `typedef` keyword

Reply:
The type is never used, so I think making it an anonymous `enum` is more appropriate.

Reply:
Just means that original code was doing something weird in multiple ways 
I personally see no point in defining a type for an enum as ordinary enumeration members have `int` as their type

Reply:
I overlooked it. This defines inputSource_e as anonymous enum variable. `typedef` is missing ...

Reply:
`inputSource_e` isn't used anywhere, so I don't think it should be a typedef.